brainstormforce/astra
# AST-1624 - fix - Blog page Meta Settings not work as it is the archive page

### Description


- On Shop/blog pages the meta settings won't work.
- As it is the archive page, can we add any notice into the Astra Meta settings.
- As users might be confused about this why the Meta settings are not working for the Blog page?

- After Discussion decided to make Disable the footer & Disable the header working & Hide the remaining option from the archive pages.

### Screenshots


### Types of changes





### How has this been tested?


1. Go To  WordPress setting → Reading → 
2. Select a static page inside that selects any page as a recent post page.
3. Check on the particular page, from there meta setting options are hidden.
4. Content Layout & sidebar options are not visible on the selected page.
5. Also, check to the disabled header & footer are working on the particular selected archive page

### Checklist:
